### DISCUSSION TITLE
Can you determine who finished where in each game?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#553] [Galaxy Trivia] [answered]
At a recent bowling match, two games were played.

Kelly beat Sam in both games, also Reed beat Jamie in both games. The winner in game 1 came second in game 2. Reed won game 2 and Jamie beat Sam in game 1. No player got the same placing twice.

**Comment by peterbaccaro** [comment, marked as answer]
![Capture](https://user-images.githubusercontent.com/74666079/231820974-c25363c8-4e52-4563-84e6-c5d8ca111b9e.JPG)

**Comment by thuongtruong1009** [comment]
Game 1:

1st place: Reed
2nd place: Jamie
3rd place: Kelly
4th place: Sam

Game 2:

1st place: Kelly
2nd place: Jamie
3rd place: Reed
4th place: Sam
Explanation:

We know that Kelly beat Sam in both games, so Kelly must have finished ahead of Sam in both games. Similarly, Reed beat Jamie in both games, so Reed must have finished ahead of Jamie in both games.

We also know that the winner in game 1 came second in game 2. So if we assume that Kelly won game 1, then she would have come second in game 2 (since she won game 2), which would violate the rule that no player got the same placing twice. Therefore, we can conclude that Reed won game 1 and came third in game 2.

We know that Jamie beat Sam in game 1, so Jamie must have finished ahead of Sam in game 1. Since Jamie came second in game 2, we can conclude that Kelly must have won game 2 (since she came first in game 1 and no player got the same placing twice), and therefore Sam must have finished last in both games.

Putting all of this information together, we get the breakdown of who finished where in each game as shown above.

**Reply by ParanoidUser** [reply]
> Similarly, Reed beat Jamie in both games, so Reed must have finished ahead of Jamie in both games.

Then why is Jamie on 2 and Reed on 3 in the second game? 😟
